tests/unit/validation/test_validation_functions.py

```diff
@@ -113,7 +121,7 @@ def test_validate_unique_ids_across_components():
     assert len(unique_id_errors[0].ids) == 4
 
 
-def test_validate_ids_exist():
+def test_validate_ids():
```

Comment:
Should this also be tested for sparse, does that make sense?

Reply:
It's almost remotely related.

Reply:
@figueroa1395 Nope, it only receives SingleDataset for both input and update.
The validation function recieves a list of single dataset for sparse / dense batch data, right?

Reply:
`validate_ids()` indeed only recieves `SingleDataset` for both input and update, hence that is fine. My questioning was more geared towards, should we implement this function for sparse as well? However, I believe this is only relevant for homogenous, so it should be fine as is, right?

Reply:
Awesomo

Reply:
Should be fine.